src/components/scenes/TransactionListScene.tsx

```diff
@@ -249,6 +252,14 @@ function TransactionListComponent(props: Props) {
   )
 }
 
+export const TransactionListTitle = () => {
+  const route = useRoute<RouteProp<'walletDetails'>>()
+  const account = useSelector(state => state.core.account)
+  const wallet = account.currencyWallets[route.params.walletId]
+  const title = wallet == null ? '' : getWalletName(wallet)
+  return <HeaderTitle title={title} />
+}
+
```

Comment:
Thanks. But now that this is here, the `walletName` and `countryCode` route props are unused. For the the `TransactionListScene`, the route props should just be `walletId`, `tokenId`, and `searchText`.